### PR TITLE
refactor: update cherry-pick for new crbug ui

### PR DESCRIPTION
**Pull request by codebytere**
Updates `crbug.js` to compensate for migration to the new Chromium issue tracker.

To determine what endpoint to hit, i dug through the [included buganizer script](https://www.gstatic.com/_/buganizer/_/js/k=buganizer.b.en_US.e8zBqVLSP44.2018.O/am=wA/d=1/rs=AI3Ihhp2ogytZehIP7x8HOr1dtgZcvyAng/m=c) and found the issue fetch endpoint.

From there, i looked at the UI to find the CVE label id: 

Screenshot 2024-02-09 at 12 05 53 PM

and reverse engineered the rest to be able to pull out the CVE again.

### CODE REVIEW

File: src/utils/crbug.js [hunk 1]
```diff
@@ -1,66 +1,79 @@
...
+      // This API call can sometimes return errant invalid characters at the start of the response.
+      const cleaned = rawJSON.substring(rawJSON.indexOf('['), rawJSON.length);
+      return JSON.parse(cleaned);
+    });
 
   return result;
 }
 
+function parseCveFromIssue(issue) {
+  const CVE_ID = 1223136;
```
Comment (ckerr) [inline src/utils/crbug.js:56]: I don't know why this is hardcoded here. What is `1223136`?

File: src/utils/crbug.js [hunk 2]
```diff
@@ -1,66 +1,79 @@
...
+
+  const issueData = issue[0][1];
+  const issueMetaData = issueData[issueData.length - 1];
+  const cveData = issueMetaData[2][14].find(d => d[0] === CVE_ID);
```
Comment (ckerr) [inline src/utils/crbug.js:60]: this code, e.g. `[2][14]` is _really_ specific and needs a code comment explainer. I probably couldn't triage a bug report against this coming in cold. :smile_cat:

**Review by ckerr — COMMENTED** [review]
I should use this feature more.

File: src/utils/crbug.js (hunk 1 above)
Comment (codebytere) [reply src/utils/crbug.js:56]: See image in PR body - it's the ID corresponding to the bit of metadata for the CVE. it's visible via the UI and also in the payload:

```json
[
                1223136,
                1363614,
                5,
                null,
                "CVE",
                null,
                null,
                [],
                2
            ]
```

Comment (ckerr) [reply src/utils/crbug.js:56]: I do see the image, sorry for not asking correctly. I get that there's a correlation here, but I don't understand what it means :laughing:

**Review by ckerr — APPROVED** [review]
I still find this code a little confusing, but there's only so much we can do here if the upstream payload is undocumented.

Morever, this feature is broken right now so it's a question of not-perfect vs broken. 

So :shrug: I'm fine with this; we can take cleanup / documentation as a followup task if needed.

**Review by ckerr — APPROVED** [review]

**Pull request merged**